### ISSUE TITLE
ListenBucketNotification doesn't show any error when wrong information are provided

### ISSUE THREAD

**Issue by vadmeste** [minio/minio-go#725]
If the server address or access or secret keys or bucket name is wrong. ListenBucketNotification doesn't return any error. This is a confusing behavior for users.

**Comment by harshavardhana** [comment]
> If the server address or access or secret keys or bucket name is wrong. ListenBucketNotification doesn't return any error. This is a confusing behavior for users.

Yes there is sort of infinite loop of sort.

**Comment by Rio** [comment]
This is also the case if tls is turned on while the server doesn't serve tls or vise versa.

**Comment by harshavardhana** [comment]
> This is also the case if tls is turned on while the server doesn't serve tls or vise versa.

Understood :+1: all errors..

**Comment by harshavardhana** [comment]
i see that listenbucket fails properly .. there is one error that needs to be fixed.

```
go run listenbucketnotification.go 
2017/11/01 01:17:49 The access key ID you provided does not exist in our records.
exit status 1
```

```
go run listenbucketnotification.go 
2017/11/01 01:22:32 Bucket name contains invalid characters
exit status 1
```

**Comment by harshavardhana** [comment]
fixed by #861
